lib/tree.rb

```diff
@@ -55,13 +72,14 @@ def tree_data
       basket << tree.pick_an_apple!
     end
 
+
     diameter_sum = 0
 
     basket.each do |apple|
       diameter_sum += apple.diameter
     end
 
-    avg_diameter = # It's up to you to calculate the average diameter for this harvest.
+    avg_diameter = diameter_sum / basket.count
```

Comment:
This seems to assume that users of the `Apple` class will always initialize it with a floating point diameter, as you did above.